pkgs/tools/misc/urjtag/default.nix

```diff
@@ -0,0 +1,39 @@
+{ pythonBindings ? false
+, jedecSupport ? false
+, python3 ? null
+, stdenv, autoconf, automake, pkgconfig, gettext, intltool, libtool, bison, flex, fetchgit, makeWrapper
```

Comment:
The general style is to have mandatory arguments first, then the optional ones. And wrap at 80 chars. (You picked the right comma-style (comma first on the next line) on first try. Good!)